models/user.model.js

```diff
@@ -0,0 +1,48 @@
+// ---------------
```

Comment:
There is already a schema file for both user and address schema so we can remove this file.